com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs

```diff
@@ -314,14 +314,15 @@ public void TestAgent()
             agent1.LazyInitialize();
 
             var numberAgent1Reset = 0;
+            var numberAgent2Reset = 0;
```

Comment:
Sorry, realize you did this in the other PR too - can you rename these variables to reflect the change in the method name (OnEpisodeBegin)?